music21/chord/__init__.py

```diff
@@ -1040,23 +1074,30 @@ def bass(self,
 
             if not foundBassInChord:  # it's not there, needs to be added
                 self.pitches = (newbass, *(p for p in self.pitches))
+            # to be changed in v8 -- do not add this bass to the chord itself.
```

Comment:
is the idea to raise an Exception?

Reply:
That's probably the best solution.  In any case it'd be nice if bass() worked very similarly to root().  There's a good reason to want to have a root that is not present in the chord or is not the normal one that is there.  To add a new bass note, all you need to do is append a note to the chord -- and if you're not sure of the octave you can use transposeBelowPitch (or whatever the name of that function is).